bazelrio/deps.bzl

```diff
@@ -52,98 +52,252 @@ def setup_bazelrio_dependencies():
+        sha256 = "5e3311ef761b4c79ebf67f78aed423d1650f3477120d4c483925e6b4c9d2eec6",
+        build_file_content = filegroup_all,
+    )
+    maybe(
+        http_archive,
+        "__bazelrio_edu_wpi_first_wpilibnewcommands_headers",
+        url = "https://frcmaven.wpi.edu/release/edu/wpi/first/wpilibNewCommands/wpilibNewCommands-cpp/2021.3.1/wpilibNewCommands-cpp-2021.3.1-headers.zip",
+        sha256 = "c919969657ce44a5a20cc84140997149082dbfda5cda7f170b68d2e94744921e",
+        build_file_content = cc_library_headers,
+    )
     maybe(
         http_archive,
         "__bazelrio_edu_wpi_first_ni-libraries_chipobject_linuxathena",
-        url = "https://frcmaven.wpi.edu/artifactory/release/edu/wpi/first/ni-libraries/chipobject/2020.9.2/chipobject-2020.9.2-linuxathena.zip",
+        url = "https://frcmaven.wpi.edu/release/edu/wpi/first/ni-libraries/chipobject/2020.9.2/chipobject-2020.9.2-linuxathena.zip",
         sha256 = "036060af780bc4d3b852d389a5210ef73df3041b9399762e7be8ea8291bb16e3",
         build_file_content = filegroup_all,
     )
     maybe(
         http_archive,
         "__bazelrio_edu_wpi_first_ni-libraries_visa_linuxathena",
-        url = "https://frcmaven.wpi.edu/artifactory/release/edu/wpi/first/ni-libraries/visa/2020.9.2/visa-2020.9.2-linuxathena.zip",
+        url = "https://frcmaven.wpi.edu/release/edu/wpi/first/ni-libraries/visa/2020.9.2/visa-2020.9.2-linuxathena.zip",
         sha256 = "2cf83ce761145ee57b1b88d817cbadcbf1b34cf1759a813fae1cd135c23e6588",
         build_file_content = filegroup_all,
     )
     maybe(
         http_archive,
         "__bazelrio_edu_wpi_first_ni-libraries_runtime_linuxathena",
-        url = "https://frcmaven.wpi.edu/artifactory/release/edu/wpi/first/ni-libraries/runtime/2020.9.2/runtime-2020.9.2-linuxathena.zip",
+        url = "https://frcmaven.wpi.edu/release/edu/wpi/first/ni-libraries/runtime/2020.9.2/runtime-2020.9.2-linuxathena.zip",
         sha256 = "a0aeff05908590b1c63071e02d635010bde54c3abb6dc016c78d5617a72c555a",
         build_file_content = filegroup_all,
     )
     maybe(
         http_archive,
         "__bazelrio_edu_wpi_first_ni-libraries_netcomm_linuxathena",
-        url = "https://frcmaven.wpi.edu/artifactory/release/edu/wpi/first/ni-libraries/netcomm/2020.9.2/netcomm-2020.9.2-linuxathena.zip",
+        url = "https://frcmaven.wpi.edu/release/edu/wpi/first/ni-libraries/netcomm/2020.9.2/netcomm-2020.9.2-linuxathena.zip",
         sha256 = "6a3dac9d74fe20358812924cc3c2bfaafde189e2b07dc444ba5d38e46ca02b06",
         build_file_content = filegroup_all,
+    )
+    maybe(
+        http_archive,
+        "__bazelrio_com_revrobotics_frc_sparkmax-cpp_linuxathena",
+        url = "http://www.revrobotics.com/content/sw/max/sdk/maven/com/revrobotics/frc/SparkMax-cpp/1.5.4/SparkMax-cpp-1.5.4-linuxathena.zip",
+        sha256 = "df0c97373fa3c033a552cf15d5b9afcd6d9231737f6bca8cf1c4df63c2d30658",
+        build_file_content = filegroup_all,
+    )
```

Comment:
I would explode this up. The compilers, NI libraries, allwpilib, and the vendors all have very different release schedules. I think versioning should be supported, at the very least for the vendor deps. 

My recommendation would be 

```
deps.bzl # calls the functions in the following subfolders
- allwplib/v2021_3_1/deps.bzl # or allwpilib.bzl, whatever
- toolchains/2021/deps.bzl # To my knowledge they never update this in season
- rev/SparkMax/v1_5_4/deps.bzl
- rev/ColorSensor/....
- ctre/Phoenix/....
```